### PR TITLE
Update htaccess.txt

### PR DESCRIPTION
**Pull request by lamjed001**
fix report.html page not found if .htaccess actived

### CODE REVIEW

File: htaccess.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ RewriteRule ^image([0-9]*).html$ do.php?img=$1
 RewriteRule ^del([a-zA-Z0-9_-]*).html$ go.php?go=del&cd=$1
 RewriteRule ^(call|guide|rules|stats).html$ go.php?go=$1
```
Comment (lamjed001) [inline htaccess.txt]: change it to
`RewriteRule ^(call|guide|rules|stats|report).html$ go.php?go=$1`

File: htaccess.txt [hunk 2]
```diff
@@ -16,6 +16,7 @@ RewriteRule ^image([0-9]*).html$ do.php?img=$1
...
 RewriteRule ^report[_-]([0-9]*).html$ go.php?go=report&id=$1
+RewriteRule ^report.html$ go.php?go=report
```
Comment (lamjed001) [inline htaccess.txt]: moved up

**Comment by abdumu** [comment]
add another commit with the new revision

**Comment by lamjed001** [comment]
sorry i don't know how to update this commit so i make new pull request
Because it's my first time i make pull request

**Comment by abdumu** [comment]
you could've easily add another commit to this branch and it will appear here. Try that next time, and thank you for the PR.

**Pull request closed without merging**